app/controllers/candidate_interface/apply_controller.rb

```diff
@@ -0,0 +1,18 @@
+module CandidateInterface
+  class ApplyController < CandidateInterfaceController
+    def show
+      @provider_code = params.fetch(:providerCode)
```

Comment:
Could we use the attributes from `@course` in the views instead?

https://thoughtbot.com/blog/sandi-metz-rules-for-developers#only-instantiate-one-object-in-the-controller

Reply:
Done